map_model/src/edits/mod.rs

```diff
@@ -606,6 +621,88 @@ fn recalculate_intersection_polygon(
     affected
 }
 
+fn fix_buildings(map: &mut Map, input: Vec<BuildingID>) {
+    // TODO Copying from make/buildings.rs
```

Comment:
There's some code duplication here and for parking lots. I refactored some of it, but I'm not sure if it's worth going any further.